### PR TITLE
rpc - list presets with tags

### PR DESCRIPTION
**Pull request by rjastrzebski**

### CODE REVIEW

File: src/vmshepherd/http/rpc_api.py [hunk 1]
```diff
@@ -57,6 +57,31 @@ def rpc_logger(func):
         presets = await self.request.app.vmshepherd.preset_manager.list_presets()
         return list(presets.keys())
 
+    @enabled_checker
+    async def list_presets_with_tags(self):
+        """ Gets more verbose list of available presets
+
+        :return:  (dict of presets with tags)
+
+        :rtype: dict of dicts
```
Comment (evemorgen) [inline src/vmshepherd/http/rpc_api.py]: If you already took effort to specify return type why not properly type annotating the whole thing? 

Someone already started type annotating in this codebase before, like [here](https://github.com/DreamLab/VmShepherd/blob/master/src/vmshepherd/presets/preset.py#L26) - maybe we could consider making it a common practice :)

I'm willing to make a PR that would type annotate the rest

File: src/vmshepherd/http/rpc_api.py [hunk 2]
```diff
@@ -57,6 +57,31 @@ def rpc_logger(func):
...
+        return {name: preset.config['meta_tags'] for name, preset in presets.items()}
+
+    @enabled_checker
+    async def get_preset_tags(self, preset):
+        """ Get tags of preset
+
+        :arg string preset: preset name
+        :return: preset tags
+
+        :rtype: dict
```
Comment (evemorgen) [inline src/vmshepherd/http/rpc_api.py]: I would appreciate if that could be more specific. I assume that would be `Dict[str, str]`?

Comment (kwarunek) [reply src/vmshepherd/http/rpc_api.py]: IMHO it won't help, especially here `Dict[str,dict]`, without an example of such a structure is....

File: src/vmshepherd/http/rpc_api.py (hunk 1 above)
Comment (rjastrzebski) [reply src/vmshepherd/http/rpc_api.py]: Its good suggestion, but its about changing code in a lot of places, so in next PR, your, my or somebody :)

File: src/vmshepherd/http/rpc_api.py (hunk 2 above)
Comment (rjastrzebski) [reply src/vmshepherd/http/rpc_api.py]: Actualy here is a simple structure like {'k1': 'v1', 'k2': 'v2'}, so I'm agree with you it should be Dict[str, str], so in PR with type annotating fixies

**Pull request merged**